packages/edit-site/src/components/plugin-template-setting-panel/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Defines an extensibility slot for the Template sidebar.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+export const { Fill, Slot } = createSlotFill( 'PluginTemplateSettingPanel' );
```

Comment:
```suggestion
const { Fill, Slot } = createSlotFill( 'PluginTemplateSettingPanel' );
```
We probably don't need to export these.